indico/modules/rb/operations/bookings_test.py

```diff
@@ -61,12 +68,34 @@ def past_booking_occurrences_are_cancelled(dummy_user, create_reservation):
         'repeat_frequency': RepeatFrequency.DAY,
         'repeat_interval': reservation.repeat_interval
     }
-    new_reservation = split_booking(
-        reservation,
-        new_booking_data,
-        extra_fields={},
-    )
-
-    number_of_cancelled_occurrences = [occ for occ in reservation.occurrences if occ.is_cancelled]
-    assert number_of_cancelled_occurrences == 2
-    assert len(new_reservation.occurrences) == 4
+
+    class MockDatetime:
+        @staticmethod
+        def now():
+            return datetime_now
+
+        @staticmethod
+        def combine(date_obj, time_obj):
+            return datetime.combine(date_obj, time_obj)
+
+    class MockDate:
+        @staticmethod
+        def today():
+            return datetime_now.date()
```

Comment:
This is not necessary, you can use the `freeze_time` fixture and then call it like this in your test to change "now" to that date/time:

`freeze_time(datetime(....))`